aict_tools/io.py

```diff
@@ -550,7 +550,7 @@ def read_cta_dl1(path, aict_config, key=None, columns=None, first=None, last=Non
         df = vstack(tel_dfs)
         if aict_config.disp:
             df = convert_units(df, aict_config.disp)
-        df = df.to_pandas()
+        df = pd.DataFrame(np.array(df)) # workaround for #11286 in astropy 4.2
```

Comment:
I think `table.data` is the correct way to get a numpy structured array from the table.

Reply:
There is no `table.data`.
But using `as_array()` might be better